File: src/targets/kotlin/okhttp/fixtures/multipart-form-data.kt
val client = OkHttpClient()

val mediaType = MediaType.parse("multipart/form-data; boundary=---011000010111000001101001")
val body = RequestBody.create(mediaType, "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--\r\n")
val request = Request.Builder()
  .url("http://mockbin.com/har")
  .post(body)
  .addHeader("Content-Type", "multipart/form-data; boundary=---011000010111000001101001")
  .build()

val response = client.newCall(request).execute()

File: src/targets/dart/http/fixtures/multipart-form-data.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'Content-Type': 'multipart/form-data; boundary=---011000010111000001101001',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
    body: "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--\r\n",
  );

  print(response.statusCode);
  print(response.body);

}